api-reference/api-reference-template.md

```diff
@@ -0,0 +1,204 @@
+# API Reference
+
+{Before using this template, read [the accompanying guide](api-reference-guide.md) to this template}.
+
+
+## Overview
+
+This reference describes the Application Program Interfaces (APIs) that help you {access | customize | program} {product}'s {features | functionality}.
+
+### Base URL
+
+The APIs referenced in this documentation use the following base URL:
+
+```
+https://api.example.com
+```
+
+
+### Authorization
+
+All requests to the API must be authenticated and authorized before they can access or manage resources on the {environment | platform}.
+
+The {product} API uses the `{auth_type}` authorization type.
+
+You can authenticate your requests by using {authentication type and method}.
+
+For example:
+
+```
+{EXAMPLE}
+```
+
+
+API requests without valid authentication will fail with a status of `{error_code}`.
+
+
+### Versioning
+
+{This section is optional.}
+
+The {product} APIs provide multiple versions of API endpoints.
+
+{Commentary about the versions supported.}
+
+### Pagination
+
+{This section is optional.}
+
+Due to the potential very large result sets from API calls, responses {are | can be} returned as shorter pages.
+
+Pagination can be customized using {pagination settings}. If not specified, the default values are {values}.
+
+### Rate limiting and throttling
+
+{This section is optional.}
+
+The {product} APIs use a {strategy_name} rate limiting strategy. The maximum number of requests allowed to access a {resource | endpoint |..} is {number} requests per {time period}.
+
+If the number of requests exceeds that limit, an HTTP `429 Too Many Request` error is returned.
```

Comment:
Remove this section or change it to cover non-standard responses. If the HTTP request is standard 429, it doesn't need to be covered in the documentation. You only need to document custom payloads or non-standard HTTP codes for error states.